src/App.tsx

```diff
@@ -20,17 +24,94 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+  };
+
+  handleWidthChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState(prevState => ({
+      ...prevState,
+      itemWidth: +event.target.value,
+    }));
+  };
+
+  handleFrameSize = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState(prevState => ({
+      ...prevState,
+      frameSize: +event.target.value,
+    }));
+  };
+
+  handleStepChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState(prevState => ({
+      ...prevState,
+      step: +event.target.value,
+    }));
+  };
+
+  handleAnimationDuration = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState(prevState => ({
+      ...prevState,
+      animationDuration: +event.target.value,
+    }));
   };
```

Comment:
You can simplify this handler and reuse for each input by adding `name`s to your inputs (or you can take `id`)
```suggestion
  handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
    const { id, value } = e.target;
    
    this.setState(prevState => ({
      ...prevState,
      [id]: +value,
    }));
  };
```